File: src/test/java/org/dedda/bratwurst/SuiteTest.java
package org.dedda.bratwurst;

import org.dedda.bratwurst.test.TestSuite;
import org.testng.annotations.Test;

import static org.testng.AssertJUnit.assertTrue;

/**
 * Created by dedda on 1/24/16.
 *
 * @author dedda
 */
public class SuiteTest {

    @Test
    public void testSuite() {
        TestSuite testSuite = new TestSuite("src/test/testSuite.bw");
        boolean success = testSuite.run();
        assertTrue("Test suite failed!", success);
    }

}

File: src/test/java/org/dedda/bratwurst/SuiteTest.kt
package org.dedda.bratwurst

import org.dedda.bratwurst.test.TestSuite
import org.testng.AssertJUnit
import org.testng.annotations.Test

/**
 * Created by dedda on 1/24/16.
 *
 * @author dedda
 */
class SuiteTest {
    @Test
    fun testSuite() {
        val testSuite = TestSuite("src/test/testSuite.bw")
        val success = testSuite.run()
        AssertJUnit.assertTrue("Test suite failed!", success)
    }
}